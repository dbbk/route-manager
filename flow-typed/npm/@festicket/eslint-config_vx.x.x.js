// flow-typed signature: 564086c0274f153b4c37ac5c954f219e
// flow-typed version: <<STUB>>/@festicket/eslint-config_v^1.1.5/flow_v0.63.1

/**
 * This is an autogenerated libdef stub for:
 *
 *   '@festicket/eslint-config'
 *
 * Fill this stub out by replacing all the `any` types.
 *
 * Once filled out, we encourage you to share your work with the
 * community by sending a pull request to:
 * https://github.com/flowtype/flow-typed
 */

declare module '@festicket/eslint-config' {
  declare module.exports: any;
}

/**
 * We include stubs for each file inside this npm package in case you need to
 * require those files directly. Feel free to delete any files that aren't
 * needed.
 */


// Filename aliases
declare module '@festicket/eslint-config/index' {
  declare module.exports: $Exports<'@festicket/eslint-config'>;
}
declare module '@festicket/eslint-config/index.js' {
  declare module.exports: $Exports<'@festicket/eslint-config'>;
}
