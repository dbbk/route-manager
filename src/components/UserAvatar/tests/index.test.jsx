import React from 'react';
import { mount } from 'enzyme';
import { UserAvatar } from '@festicket/react-ui-components';

const avatarLarge = 'http://via.placeholder.com/350x150';
const initial = 'B';

describe('<UserAvatar />', () => {
  test('with initial props', () => {
    const tree = mount(<UserAvatar />);
    expect(tree).toMatchSnapshot();
  });

  test('with avatarLarge', () => {
    const tree = mount(<UserAvatar avatarLarge={avatarLarge} />);
    expect(tree).toMatchSnapshot();
  });

  test('with initial', () => {
    const tree = mount(<UserAvatar initial={initial} />);
    expect(tree).toMatchSnapshot();
  });
});
