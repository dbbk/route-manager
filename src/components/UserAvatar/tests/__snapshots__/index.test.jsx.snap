// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<UserAvatar /> with avatarLarge 1`] = `
.c0 {
  box-sizing: border-box;
  border-radius: 50%;
  display: -webkit-inline-box;
  display: -webkit-inline-flex;
  display: -ms-inline-flexbox;
  display: inline-flex;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  -webkit-box-pack: center;
  -webkit-justify-content: center;
  -ms-flex-pack: center;
  justify-content: center;
  overflow: hidden;
  font-size: 12px;
  width: 24px;
  height: 24px;
}

.c1 {
  width: 100%;
  height: 100%;
  object-fit: cover;
}

@media (min-width:569px) {
  .c0 {
    font-size: 16px;
    width: 44px;
    height: 44px;
  }
}

<span
  className="c0"
>
  <img
    alt=""
    className="c1"
    src="http://via.placeholder.com/350x150"
  />
</span>
`;

exports[`<UserAvatar /> with avatarLarge and large variant 1`] = `
.c0 {
  box-sizing: border-box;
  border-radius: 50%;
  display: -webkit-inline-box;
  display: -webkit-inline-flex;
  display: -ms-inline-flexbox;
  display: inline-flex;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  -webkit-box-pack: center;
  -webkit-justify-content: center;
  -ms-flex-pack: center;
  justify-content: center;
  overflow: hidden;
  font-size: 25px;
  width: 88px;
  height: 88px;
}

.c1 {
  width: 100%;
  height: 100%;
  object-fit: cover;
}

<span
  className="c0"
>
  <img
    alt=""
    className="c1"
    src="http://via.placeholder.com/350x150"
  />
</span>
`;

exports[`<UserAvatar /> with avatarLarge and medium variant 1`] = `
.c0 {
  box-sizing: border-box;
  border-radius: 50%;
  display: -webkit-inline-box;
  display: -webkit-inline-flex;
  display: -ms-inline-flexbox;
  display: inline-flex;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  -webkit-box-pack: center;
  -webkit-justify-content: center;
  -ms-flex-pack: center;
  justify-content: center;
  overflow: hidden;
  font-size: 23px;
  width: 52px;
  height: 52px;
}

.c1 {
  width: 100%;
  height: 100%;
  object-fit: cover;
}

<span
  className="c0"
>
  <img
    alt=""
    className="c1"
    src="http://via.placeholder.com/350x150"
  />
</span>
`;

exports[`<UserAvatar /> with name 1`] = `
.c0 {
  box-sizing: border-box;
  border-radius: 50%;
  display: -webkit-inline-box;
  display: -webkit-inline-flex;
  display: -ms-inline-flexbox;
  display: inline-flex;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  -webkit-box-pack: center;
  -webkit-justify-content: center;
  -ms-flex-pack: center;
  justify-content: center;
  overflow: hidden;
  border: 2px solid #8C64CB;
  color: #8C64CB;
  -webkit-text-transform: capitalize;
  text-transform: capitalize;
  -webkit-text-align: center;
  text-align: center;
  font-weight: 700;
  font-size: 12px;
  width: 24px;
  height: 24px;
}

@media (min-width:569px) {
  .c0 {
    font-size: 16px;
    width: 44px;
    height: 44px;
  }
}

<span
  className="c0"
>
  B
</span>
`;

exports[`<UserAvatar /> with name and large variant 1`] = `
.c0 {
  box-sizing: border-box;
  border-radius: 50%;
  display: -webkit-inline-box;
  display: -webkit-inline-flex;
  display: -ms-inline-flexbox;
  display: inline-flex;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  -webkit-box-pack: center;
  -webkit-justify-content: center;
  -ms-flex-pack: center;
  justify-content: center;
  overflow: hidden;
  border: 2px solid #8C64CB;
  color: #8C64CB;
  -webkit-text-transform: capitalize;
  text-transform: capitalize;
  -webkit-text-align: center;
  text-align: center;
  font-weight: 700;
  font-size: 25px;
  width: 88px;
  height: 88px;
}

<span
  className="c0"
>
  B
</span>
`;

exports[`<UserAvatar /> with name and medium variant 1`] = `
.c0 {
  box-sizing: border-box;
  border-radius: 50%;
  display: -webkit-inline-box;
  display: -webkit-inline-flex;
  display: -ms-inline-flexbox;
  display: inline-flex;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  -webkit-box-pack: center;
  -webkit-justify-content: center;
  -ms-flex-pack: center;
  justify-content: center;
  overflow: hidden;
  border: 2px solid #8C64CB;
  color: #8C64CB;
  -webkit-text-transform: capitalize;
  text-transform: capitalize;
  -webkit-text-align: center;
  text-align: center;
  font-weight: 700;
  font-size: 23px;
  width: 52px;
  height: 52px;
}

<span
  className="c0"
>
  B
</span>
`;
